contents/handbook/growth/marketing/index.md

```diff
@@ -124,20 +70,38 @@ For these reasons, we:
  
 - Don't pretend our customers are different from us – i.e. more gullible, more susceptible to marketing. We are an engineering-led team building products for other engineers. If you wouldn't like it, assume our customers wouldn't either. 
 
-## Marketing channels and tactics today
+## Marketing vision 
 
-The Marketing team's main focus is acquisition of customers in our [ideal customer profile](/handbook/strategy/ideal-customer-persona) (ICP). 
+### Things we want to be brilliant at
+ 
+- **Word of mouth mindset:** By IPO, we want to have built a hugely successful company driven primarily by word of mouth, rather than paid ads or PR.
```

Comment:
```suggestion
- **Word of mouth mindset:** By IPO, we want to build a hugely successful company driven primarily by word of mouth, rather than paid ads or PR.
```